packages/core/src/FrameLoop.ts

```diff
@@ -159,7 +159,7 @@ export class FrameLoop {
         : config.initialVelocity
 
       const precision =
-        config.precision || Math.min(1, Math.abs(to - from) * 0.001)
+        config.precision || Math.min(1, Math.abs(to - from) * 0.001) || 0.005
```

Comment:
Animations enter the frameloop when `to` and `from` are equal

Reply:
Ah nevermind actually, this will be possible with #807 because `to` can be a dependency.